grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -673,12 +675,14 @@ class NotifyService implements SendgridSupport, Status {
         sendMail(mail, getApiKey(), getSendGridUrl())
     }
 
-    Map<Boolean, String> sendEditsDisapprovedNotification(Issue issue) {
+    Map<Boolean, String> sendEditsDisapprovedNotification(Issue issue, String editCreatorName) {
         String type = issue.type?.equals(IssueType.CONSENT_GROUP.getName()) ? "Consent Group" : "Project"
+        User editCreator = userService.findUser(editCreatorName)
```

Comment:
Ditto here.

Reply:
Solved in a similar way as clarificationRequest Controller -> 
https://github.com/broadinstitute/orsp-pub/pull/188/files#diff-0401de3e18519430c4c05ee635382da1R666